pkgs/by-name/me/meow/Cargo.toml.patch

```diff
@@ -0,0 +1,5 @@
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -2 +2 @@
+-name = "meow-cli"
++name = "meow"
```

Comment:
What's the reason of this?

Reply:
I want the command-line application to be run using the `meow` command, not using the `meow-cli` command.
I could not find a way to do this without patching the Cargo.toml file; changing it in the original file would not work since the Cargo package name `meow` is already taken and I would not be able to run `cargo publish`.

If you have a better solution to this, I would really appreciate it :)

Reply:
```
  postInstall = ''
    ln -s $out/bin/meow-cli $out/bin/meow
  '';
```

Also you are the upstream so you decide whatever upstream ships, but don't do dirty tricks here

Reply:
Omg thanks, I was pulling my hair out cause I couldn't get `postInstall` to work. This works now :D